__test__/SearchReview.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import SearchReview from '../client/components/SearchReview.jsx';
+import App from '../client/components/App.jsx';
+
+
+describe('Unit Tests', () => {
+  describe('Render Tests', () => {
+    test('It should render a form', () => {
+      const wrapper = shallow(
+        <SearchReview
+          searchInputHandle = { () => {}}
+          clearField = { () => {} }
+          searchPhrase = { '' }
+        />
+      )
+      expect(wrapper.find('SearchForm')).toHaveLength(1);
+    });
+  });
+
+  describe('Component Interaction Tests', () => {
+    test('It should invoke correct method when a change event is occurred' , () => {
+      const mockSearchHandle = jest.fn();
+      const wrapper = shallow(
+        <SearchReview
+          searchInputHandle = { () => {}}
+          clearField = { () => {} }
+          searchPhrase = { '' }
+        />
+      )
+      wrapper.instance().searchHandle = mockSearchHandle;
+      wrapper.instance().forceUpdate();
+      wrapper.find('SearchInput').simulate('change', {target: { name: 'search', value:'test'}} );
+      expect(mockSearchHandle).toHaveBeenCalled();
+    });
+
```

Comment:
For the tests below this comment, I thought the article you linked said to not test state change. Not saying I agree with the article, just a thought!